app/components/primer/local_time.rb

```diff
@@ -48,5 +48,11 @@ def initialize(datetime:, weekday: DEFAULT_TEXT_TYPE, year: DEFAULT_DIGIT_TYPE,
       @system_arguments[:second] = fetch_or_fallback(DIGIT_TYPE_OPTIONS, second, DEFAULT_DIGIT_TYPE)
       @system_arguments[:"time-zone-name"] = fetch_or_fallback(TEXT_TYPE_OPTIONS, time_zone_name, DEFAULT_TEXT_TYPE)
     end
+
+    def call
+      render(Primer::BaseComponent.new(**@system_arguments)) do
+        content || @datetime.strftime("%B %-d, %Y %H:%M %Z")
+      end
```

Comment:
Should this component accept rich content, or should it always be a string? If it can only be a string, maybe we could simplify this to be:

```suggestion
      render(Primer::BaseComponent.new(**@system_arguments).with_content(@text || @datetime.strftime("%B %-d, %Y %H:%M %Z"))
```

(Adding an optional `text` argument to the component)

Reply:
I was thinking rich content but I think most cases would do strings. With that in mind, I think it makes sense to start with a string and expand to accept rich content if we have that requirement in the future.